docs/resources/excel-samples.md

```diff
@@ -17,47 +17,100 @@ The following samples are simple scripts for you to try on your own workbooks. T
+
+  // Get the used range in the worksheet.
+  let range = selectedSheet.getUsedRange();
+
+  // Set the fill color to green for the top 10% of values in the range.
+  let conditionalFormat = range.conditionalFormats.add(Excel.ConditionalFormatType.topBottom);
+  conditionalFormat.topBottom.format.fill.color = "green";
+  conditionalFormat.topBottom.rule = { 
+    rank: 10, // The percentage threshold.
+    type: Excel.ConditionalTopBottomCriterionType.topPercent // The type of the top/bottom condition.
+  };
+}
+```
+
+### Create a sorted table
 
 This sample creates a table from the current worksheet's used range, then sorts it based on the first column.
 
 ```TypeScript
 async function main(context: Excel.RequestContext) {
-    // Get the current worksheet.
-    let workbook = context.workbook;
-    let worksheets = workbook.worksheets;
-    let selectedSheet = worksheets.getActiveWorksheet();
+  // Get the current worksheet.
+  let selectedSheet = context.workbook.worksheets.getActiveWorksheet();
 
-    // Create a table with the used cells.
-    let usedRange = selectedSheet.getUsedRange();
-    let newTable = selectedSheet.tables.add(usedRange, true);
+  // Create a table with the used cells.
+  let usedRange = selectedSheet.getUsedRange();
+  let newTable = selectedSheet.tables.add(usedRange, true);
 
-    // Sort the table using the first column.
-    newTable.sort.apply([{ key: 0, ascending: true }]);
+  // Sort the table using the first column.
+  newTable.sort.apply([{ key: 0, ascending: true }]);
 }
 ```
 
-## Delete resolved comments
+## Collaboration
+
+These samples help your script use the collaboration tools in Excel, such as comments.
+
+### Delete resolved comments
 
 This sample deletes all resolved comments from the current worksheet.
 
 ```TypeScript
 async function main(context: Excel.RequestContext) {
-    // Get the current worksheet.
-    let workbook = context.workbook;
-    let worksheets = workbook.worksheets;
-    let selectedSheet = worksheets.getActiveWorksheet();
-
-    // Get the comments on this worksheet.
-    let comments = selectedSheet.comments;
-    comments.load("items/resolved");
-    await context.sync();
-
-    // Delete the resolved comments.
-    comments.items.forEach((comment) => {
-        if (comment.resolved) {
-            comment.delete();
-        }
-    });
+  // Get the current worksheet.
+  let selectedSheet = context.workbook.worksheets.getActiveWorksheet();
+
+  // Get the comments on this worksheet.
+  let comments = selectedSheet.comments;
+  comments.load("items/resolved");
+  await context.sync();
+
+  // Delete the resolved comments.
+  comments.items.forEach((comment) => {
+      if (comment.resolved) {
+          comment.delete();
+      }
+  });
 }
 ```
+
+## Suggest new samples
+
+We welcome suggestions for new samples. If there is a common scenario that would help other script developers, please tell us in the feedback section below.
```

Comment:
FYI -- looks like the feedback control isn't yet enabled for the Office Script docs -- i.e., it's not present on the live site -- https://docs.microsoft.com/office/dev/scripts/resources/excel-samples -- because of this setting in the **docfx.json** file:

![image](https://user-images.githubusercontent.com/3019529/72182527-848f7700-33a0-11ea-9407-e81b36620a00.png)

Instructions for enabling it are here in the OPS docs:  https://review.docs.microsoft.com/en-us/help/onboard/github-feedback?branch=master#configure-the-feedback-control-in-your-repository

Essentially, you just need to edit **docfx.json** to replace the highlighted line in the screenshot above with these 3 lines:

```
"feedback_system": "GitHub",
"feedback_github_repo": "OfficeDev/office-scripts-docs",
"feedback_product_url": "https://excel.uservoice.com/forums/274580-excel-for-the-web?category_id=143439"
```

Maybe you can include this update in this PR as well?